inlineplz/linter_runner.py

```diff
@@ -0,0 +1,332 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=W0703,C0412
+"""Linter configurations."""
+
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import asyncio
+import fnmatch
+import multiprocessing
+import os.path
+import shutil
+import sys
+import time
+import traceback
+
+from identify import identify
+import inlineplz.linters
+from inlineplz import message
+from inlineplz import registry
+from inlineplz.util import system
+
+
+class LinterRunner:
```

Comment:
[inline-plz]: `prospector: pylint: Too many instance attributes (12/7) (too-many-instance-attributes)`

Reply:
im fine with ignoring this for now